subsys/nrf_security/src/drivers/cracen/common/include/cracen/lib_kmu.h

```diff
@@ -64,9 +64,9 @@ struct kmu_src_t {
 	/** Revocation policy. */
 	uint32_t rpolicy;
 	/** 32-bit destination address. Cannot point to SICR and must be on a
-	 * 64-bit boundary.
+	 * 128-bit boundary.
```

Comment:
I guess here we can say that is 64 bit boundary for the pdk and 128 for the dk, right?